core/api/serializers/cp_report.py

```diff
@@ -222,17 +202,24 @@ def create(self, validated_data):
 
         cp_report = cp_report_serializer.save()
 
-        self._create_cp_records(cp_report, validated_data.get("section_a", []), "A")
+        self._create_cp_records(cp_report, validated_data.get("section_a", []),
+                                "A")
         self._create_prices(cp_report, validated_data.get("section_c", []))
 
         if cp_report_data["year"] > IMPORT_DB_MAX_YEAR:
-            self._create_cp_records(cp_report, validated_data.get("section_b", []), "B")
-            self._create_generation(cp_report, validated_data.get("section_d", []))
-            self._create_emission(cp_report, validated_data.get("section_e", []))
+            self._create_cp_records(cp_report,
```

Comment:
Even if it's not enforced in the github actions, the code looks like it's using black-style formatting.

Reply:
Yeah, I think I forgot to set pycharm's formatting to black here, my bad.